### PR TITLE
Add git commit id to app for #362

### PR DESCRIPTION
**Pull request by soulBit**

### CODE REVIEW

File: src/app/components/Footer/index.tsx [hunk 1]
```diff
@@ -63,6 +64,15 @@ export function Footer() {
...
         </div>
+        <div className="text-center small text-lightGrey">
+          <a
+            href={`https://github.com/DistributedCollective/Sovryn-frontend/commit/${commitHash}`}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {commitHash}
+          </a>
+        </div>
```
Comment (creed-victor) [inline src/app/components/Footer/index.tsx:75]: Maybe add condition so it wouldn't show if there is no commit id? `{commitHash && <div>...</div>}`

File: src/app/components/Footer/index.tsx [hunk 2]
```diff
@@ -63,6 +64,15 @@ export function Footer() {
             </p>
           </div>
         </div>
+        <div className="text-center small text-lightGrey">
+          <a
+            href={`https://github.com/DistributedCollective/Sovryn-frontend/commit/${commitHash}`}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {commitHash}
```
Comment (creed-victor) [inline src/app/components/Footer/index.tsx:73]: Your suggestion in issue is valid, lets keep it short like github does:
```suggestion
            {commitHash.substr(0, 7)}
```

**Comment by creed-victor** [comment]
@kayalight any suggestion how / where to put this so it would look more nicely?

User should be able to find it easily, code in screenshot is too long - will be exactly 7 characters.
It's clickable and I think having it prefixed with something like "Build id"/"Build version" or etc would also help user to find it when asked

Screenshot 2021-02-13 at 21 46 11

**Comment by kayalight** [comment]
what is this exactly communicating, the version of the dapp?
@creed-victor

**Comment by creed-victor** [comment]
> what is this exactly communicating, the version of the dapp?
> @creed-victor

Yes, though "build id" would be more correct term in this case

File: src/app/components/Footer/index.tsx (hunk 1 above)
Comment (soulBit) [reply src/app/components/Footer/index.tsx]: Good idea, no need to render the container elements if the commit id isn't present

File: src/app/components/Footer/index.tsx (hunk 2 above)
Comment (soulBit) [reply src/app/components/Footer/index.tsx]: Updated now

**Comment by soulBit** [comment]
@kayalight @creed-victor how about just a simple "build" before the id?

image

Will add the text to translations if you're happy with this

**Comment by creed-victor** [comment]
> @kayalight @creed-victor how about just a simple "build" before the id?
> 
> image
> 
> Will add the text to translations if you're happy with this

Yes, "build" should be ok, just not sure about font and styling, maybe aligning it to the left, adding same styles as other footer texts and making larger space between these elements should do the trick

**Comment by kayalight** [comment]
Screenshot 2021-02-14 at 13 46 17
[build.xd.zip](https://github.com/DistributedCollective/Sovryn-frontend/files/5977802/build.xd.zip)
@creed-victor 
attached is a file with the text styling for the footer section with build ID

**Comment by kayalight** [comment]
[build.xd.zip](https://github.com/DistributedCollective/Sovryn-frontend/files/5977807/build.xd.zip)
@creed-victor

**Comment by soulBit** [comment]
Thanks @kayalight, I like it! If @creed-victor is fine with this, I'll implement it tomorrow

**Comment by creed-victor** [comment]
> Thanks @kayalight, I like it! If @creed-victor is fine with this, I'll implement it tomorrow

Yep, can be implemented

**Review by creed-victor — APPROVED** [review]

**Pull request merged**